src/main/java/seedu/address/model/delivery/DeliveryWrapper.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.delivery;
```

Comment:
Is a wrapper class necessary? Would passing the delivery and index straight into AddDeliveryCommand make for simpler code?

Reply:
Oh I initially thought it would be weird to pass in two arguments to the AddDeliveryCommand and that a wrapper class would make more sense